lightly/__init__.py

```diff
@@ -74,8 +74,8 @@
 # Copyright (c) 2020. Lightly AG and its affiliates.
 # All Rights Reserved
 
-__name__ = 'lightly'
-__version__ = '1.1.22'
+__name__ = "lightly"
+__version__ = "1.2.0"
```

Comment:
We should decide for the whole project whether we want to do double quotes or single quotes.